dotnet/src/SemanticKernel.Abstractions/PromptTemplate/PromptTemplateConfig.cs

```diff
@@ -103,6 +104,18 @@ public class InputParameter
         public bool IsRequired { get; set; } = true;
     }
 
+    /// <summary>
+    /// Return the input parameters metadata.
+    /// </summary>
+    internal List<KernelParameterMetadata> GetKernelParametersMetadata()
```

Comment:
nit: IEnumerable<KernelParameterMetadata>?

Reply:
Will fix as part of my next PR